spacecmd/src/spacecmd/system.py

```diff
@@ -2057,6 +2057,45 @@ def do_system_rename(self, args):
 ####################
 
 
+def help_system_refreshpillar(self):
+    print(_('system_refreshpillar: Refresh pillar data of system(s) '))
+    print(_('usage: system_refreshpillar [<SYSTEMS>]'))
+    print('')
+    print(self.HELP_SYSTEM_OPTS)
+
+def complete_system_refreshpillar(self, text, line, beg, end):
+    return self.tab_complete_systems(text)
+
+def do_system_refreshpillar(self, args):
+    arg_parser = get_argument_parser()
+
+    (args, _options) = parse_command_arguments(args, arg_parser)
+
```

Comment:
Shouldn't it be checked if no system arg was passed so the help is shown in that case?

Reply:
"no arg" is a valid case. This means we do it for all system.